### DISCUSSION TITLE
Am I really using Jupyter?

### DISCUSSION THREAD

**Discussion by velle** [bernhard-42/vscode-ocp-cad-viewer#109] [Q&A]
Regarding: https://github.com/bernhard-42/vscode-ocp-cad-viewer?tab=readme-ov-file#running-code-using-jupyter. Under "Usage" the first subsection is "Running code using Jupyter". I followed the instructions, and I'm able to run my CadQuery code. 

I don't think I have ever used Jupyter before, so maybe I don't understand it properly. But I thought Jupyter was about storing Python code / snippets in the cloud instead of locally, and even running / editing them via browser... Or at least something like that. 

But as far as I can tell, my code is stored locally on my machine, and only there. Perhaps the section has a misleading name? Or have I misunderstood what Jupyter is?

Or maybe the viewer is somehow running on Jupyter?

**Comment by bernhard-42** [comment]
Jupyter is a so called notebook system, and runs code either locally or in cloud.
Basically, the notebook is the user interface and in each cell Python is used to execute the code.
VSCode has a Jupyter mode, and this is used in the viewer (running jupyter cells in vs code instead of Jupyter)

**Reply by velle** [reply]
Got it... I think. Thanks :)
